product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "EDB Postgres Distributed 5.5.0 release notes"
+navTitle: "Version 5.5.0"
+---
+
+Released: TBD
+
+EDB Postgres Distributed version 5.5.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.5. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+
+## Highlights of EDB Postgres Distributed 5.5.0
+
+Highlights of this 5.5.0 release include:
+
+* read scalability enhancements in PGD Proxy which allow read-only queries to be routed to nodes that are not the write leader, improving the overall performance of the cluster.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres
+Advanced Server or EDB Postgres Extended upgrades with an upgrade of EDB
+Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description                                   | Addresses |
+|-----------|---------|-----------------------------------------------|-----------|
+| BDR       | 5.5.0   | Add support for readonly proxy routing        |           |
+| CLI       | 5.5.0   | Create PGD CLI binaries for MacOS and Windows |           |
```

Comment:
@Jagdish-Kewat-EDB @smaher-edb Are these already available to customers? This branch adds installation instructions for Mac, but I'm not seeing anything for Windows, yet.

Reply:
According to @smaher-edb, MacOS is available (as per the documented installation instruction), but the binaries for Windows are not yet available to customers. Please remove the "and Windows" addition.